packages/perseus-score/src/widgets/matrix/validate-matrix.ts

```diff
@@ -21,10 +21,10 @@ function validateMatrix(userInput: PerseusMatrixUserInput): ValidationResult {
 
     for (let row = 0; row < suppliedSize[0]; row++) {
         for (let col = 0; col < suppliedSize[1]; col++) {
-            if (
-                supplied[row][col] == null ||
-                supplied[row][col].toString().length === 0
-            ) {
+            // The row could be undefined if the user has skipped a row.
+            const rowData = supplied[row];
+            const cellValue = rowData?.[col];
```

Comment:
100000% optional. Just depends on what feels easier to read to you.

```suggestion
            const cellValue = supplied[row]?.[col];
```